docs/layouts/shortcodes/generated/dynamic_section.html

```diff
@@ -8,6 +8,18 @@
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td><h5>kubernetes.operator.cluster.health-check.completed-checkpoints.enabled</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to enable completed checkpoint health check for clusters.</td>
+        </tr>
+        <tr>
+            <td><h5>kubernetes.operator.cluster.health-check.completed-checkpoints.window</h5></td>
```

Comment:
maybe it would be better to call it `checkpoint-progress`, and make it more explicit in the description that we record unhealthy state if no checkpoints were taken at all in the check window

Reply:
Changed + fixed other issues in the doc.